mkdocs.yml

```diff
@@ -86,5 +86,6 @@ nav:
     - elementaloperatorchart-reference.md
     - kubernetesversions.md
     - smbios.md
+    - cluster-reference.md
```

Comment:
I would move up the cluster-reference, just after the machineinventoryselectortemplate-reference.md

Reply:
Good point, I forgot this order is the actual column order in the page!